source/envs/env_avoid_observber.py

```
@@ -0,0 +1,609 @@
+
+            pygame.display.flip()
+            self.clock.tick(16)
+
+            if self.record:
+                # 현재 화면 캡처 (RGB)
+                frame = pygame.surfarray.array3d(self.screen)
+                frame = frame.swapaxes(0, 1)  # (W, H, C) → (H, W, C)
+                frame = cv2.cvtColor(frame, cv2.COLOR_RGB2BGR)
+                self.video_writer.write(frame)
+
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)